blocks/library/table/style.scss

```diff
@@ -1,9 +1,12 @@
 .wp-block-table {
 	overflow-x: auto;
 	display: block;
+	border-collapse: collapse;
+	width: 100%;
 
-	table {
-		border-collapse: collapse;
-		width: 100%;
+	td,
+	th {
+		padding: 0.5em;
+		border: 1px solid currentColor;
```

Comment:
Thinking that in general themes already have styles for tables. So I'm wondering if we should enforce this style here?

Reply:
It's a fine line. We decided not to do it for blockquotes, but we still do it for pullquotes since that's a new block.

I feel that in this case, it makes sense for the Gutenberg table block to provide some builtin styles that are _easy_ for the theme to override, which they are in this case. 

Was there a previous ticket where we removed these styles?